### PR TITLE
Remove signing server support

### PR DESCRIPTION
**Pull request by catlee**

### CODE REVIEW

File: signingscript/Dockerfile [hunk 1]
```diff
@@ -4,7 +4,7 @@ RUN groupadd --gid 10001 app && \
     useradd -g app --uid 10001 --shell /usr/sbin/nologin --create-home --home-dir /app app
 
 RUN apt-get update \
- && apt-get install -y zipalign default-jdk-headless osslsigncode \
+ && apt-get install -y zipalign osslsigncode \
```
Comment (escapewindow) [inline signingscript/Dockerfile:7]: Hm, did we need this for jar signing before, and no longer?

File: signingscript/config_example.json [hunk 2]
```diff
@@ -2,12 +2,11 @@
     "signing_server_config": "server_config.json",
     "work_dir": "/tmp/work_dir",
     "artifact_dir": "/tmp/artifact_dir",
-    "signtool": "signtool",
```
Comment (escapewindow) [inline signingscript/config_example.json:5]: \o/

Comment (escapewindow) [reply signingscript/config_example.json:5]: We may be able to delete `ssl_cert` as well; I think that's for the signing servers.

**Review by escapewindow — APPROVED** [review]

File: signingscript/Dockerfile (hunk 1 above)
Comment (catlee) [reply signingscript/Dockerfile:7]: I think maybe we used it for zipalign before it was available as its own package?

File: signingscript/config_example.json (hunk 2 above)
Comment (catlee) [reply signingscript/config_example.json:5]: Good catch! I've removed ssl_cert in a followup.

**Pull request merged**